pkg/vdf/readme.md

```diff
@@ -0,0 +1,20 @@
+# Go Bindings for chiavdf
+
+This module depends on cgo and c bindings from chiavdf. To compile, you will need to provide paths to the libraries to
+link against.
+
+This example links gmp statically so the destination system doesn't need to have it installed:
+
+```shell
+# This path needs to include the c_wrapper.h file from chiavdf's c_bindings
+export CGO_CFLAGS="-I/path/to/chiavdf/src/c_bindings"
+export CGO_LDFLAGS="-L/path/to/chiavdf/build/lib/static -L/opt/homebrew/opt/gmp/lib /opt/homebrew/opt/gmp/lib/libgmp.a"
```

Comment:
Is there some documentation about how to install gmp, or where it might be on certain systems? Maybe fair to make people look this up for themselves, I just noticed telltale signs of macos here.